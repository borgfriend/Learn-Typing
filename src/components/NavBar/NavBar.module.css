.navbar {
    width: 100%;
    height: 5rem;
    background-color: white;
    padding: 0 2rem;
    box-shadow: 0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
}