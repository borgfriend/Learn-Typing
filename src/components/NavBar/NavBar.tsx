import * as React from 'react';
import styles from './NavBar.module.css';

export const NavBar: React.FC = ({children}) => {
    return <div className={styles.navbar}>
        {children}
    </div>
}