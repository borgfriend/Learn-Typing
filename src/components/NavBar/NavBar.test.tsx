import { render } from "@testing-library/react";
import { expect, it } from "vitest";
import { NavBar } from "./NavBar";
it("renders a default button", () => {
  const { container } = render(<NavBar></NavBar>);
  expect(container).toMatchSnapshot();
});
