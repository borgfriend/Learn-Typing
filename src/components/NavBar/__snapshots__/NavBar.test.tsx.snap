// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a default button 1`] = `
<div>
  <div
    class="sc-bczRLJ hfIZzH"
  />
</div>
`;
